node/src/reactor/validator/tests.rs

```diff
@@ -50,8 +50,8 @@ impl TestChain {
             })
             .collect();
         // TODO: This is duplicated. Remove the `HighwayConfig` field.
-        chainspec.genesis.timestamp = Timestamp::now();
-        chainspec.genesis.highway_config.genesis_era_start_timestamp = Timestamp::now();
+        chainspec.genesis.timestamp = Timestamp::now() + 30000.into();
```

Comment:
Can you add a comment about the 3000?

Reply:
Done. I tried with 10 seconds and surprisingly (it's a unit test after all!) that wasn't enough!
Some `println!` debugging showed that it actually takes roughly 20 seconds until the last node starts up, so I made it 30 to avoid flakiness.